README.md

```diff
@@ -43,7 +43,62 @@ There are two types of workflows you can use to contribute documentation:
 -   [Directly in Github](#Github): Easy method if you just want to update text.
 -   [Using Gitpod](#Gitpod): Recommended if you can use VS code and want to edit more comfortably.
 
+### Before you contribute: Docs workflow
+
+This repository has at least 2 branches at all times: `main` and `docs`. 
+The Chronologue website is created and maintained on the `main` branch. 
+
+<img width="763" alt="image" src="https://user-images.githubusercontent.com/13576110/172500916-bad816c6-05f9-416b-84a5-4bbe54da3e85.png">
+
+The documentation for Chronologue is created and maintained on the `docs` branch. Since merging the `docs` branch into the `main` branch would delete the whole Chronologue website, we use branch protection rules which are visible to administrators in the repo settings. Because of these branch protection rules, you can't directly commit onto the `docs` branch. The general authoring workflow looks like this: 
+
+1. Create a new branch
+2. Make your changes, for example: create a draft.
+3. Open a pull request, so others can review your content. 
+4. When you and your reviewers agree that your documentation is ready for publication, you can merge your branch with `docs`. This requires at least **one** approval.
+   > ✔️ Once your PR is merged, your documentation is online.
 
 ### Contributing via Github<a id="Github"></a>
 
+Writing or editing files directly on Github is a good way to get started if you haven't worked with the command line or IDEs or just want to make a quick change. 
+
+#### Creating a new branch:
+
+1. Pick an issue you want to work on.
+2. Use _Create a branch_ to create a new branch that is linked to the issue. <br/>
+   <img width="252" alt="image" src="https://user-images.githubusercontent.com/13576110/172503283-338f72a5-29c4-4131-b159-040a35c5cffe.png">
+3. Change the branch source to `docs` and name the branch how you prefer. The default is the number and name of the issue.<br/>
+   <img width="500" alt="branch" src="https://user-images.githubusercontent.com/13576110/172506197-58ffe5c5-d356-444b-aa6d-a9629897350e.png">
+   > ℹ️ After you click _Create branch_, a new window pops up with information that you can use in your local development setup. Since you want to work on the website, just dismiss this window. 
+4. Click on the branch to switch to you working branch. <br/>
+   <img width="332" alt="image" src="https://user-images.githubusercontent.com/13576110/172505635-c3197f8c-eef2-42a5-9297-766e34e85b81.png">
+
+
+#### To edit a file in Github: 
+
+1. In your working branch, locate the file you want to edit. 
+2. Verify that you are in your working branch. <br/>
+   <img width="600" alt="branch-overview" src="https://user-images.githubusercontent.com/13576110/172507864-b76f3163-f084-48cc-831e-233779c76934.png">
+3. Hit the _Edit this file_ button.<br/>
```

Comment:
We use the Google Developer's Style Guide, which advocates for "click" rather than "hit" here. https://developers.google.com/style/word-list#letter-h